skore/src/skore/utils/_progress_bar.py

```diff
@@ -43,6 +43,7 @@ def wrapper(*args, **kwargs):
                     ),
                     TextColumn("[orange1]{task.percentage:>3.0f}%"),
                     expand=False,
+                    transient=True
```

Comment:
It would be what `ruff` would do ;)

```suggestion
                    transient=True,
```